odoo/tools/misc.py

```diff
@@ -1156,6 +1160,37 @@ def formatLang(env, value, digits=None, grouping=True, monetary=False, dp=False,
             res = '%s %s' % (currency_obj.symbol, res)
     return res
 
+def formatLangDate(env, value, lang_code=False, date_format=False):
+    '''
+        Formats the date in a given format.
+
+        :param env: an environment.
+        :param value: date, datetime or string. The date to format.
+        :param lang_code: string. The lang code, if not specified it is extracted from the
```

Comment:
incorrect formatting of the docstring, type should be either in the param tag, or provided via a separate type tag.